qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -625,3 +630,56 @@ def resize_columns(self):
 
     def update_icon_size(self, size):
         self.listViewResources.setIconSize(QSize(size, size))
+
+    def setup_resource_type_tree(self):
+        """
+        Setup the resource type tree widget with available resource types.
+        The tree will have a main "Resource Types" item with child items for each resource type.
+        """
+        # Clear the tree widget
+        self.treeWidgetCategories.clear()
+        
+        # Set up the header
+        self.treeWidgetCategories.setHeaderLabel("Resource Types")
+        
+        # Create the root item
+        self.treeWidgetCategories.invisibleRootItem()
+        
+        # Add resource type categories
+        categories = {
```

Comment:
Can you move it to constants.py, for example:

```python
ResoureTypeCategories = {
    ResoureType.Model: "Models",
    ResoureType.Style: "Styles",
    ResoureType.Geopackage: "Geopackages",
    ResoureType.Model3D: "3D Models",
    ResoureType.LayerDefinition: "Layer Definitions",
}
```

then you iterate the `ResoureTypeCategories`.